tracks/00_var_speed_limit_100.json

```diff
@@ -47,5 +47,17 @@
                 0.0
             ]
         ]
+    },
```

Comment:
I would not add artificially curvature = 0. If the field does not exist, the code should infer that it is equal to zero.